applications/konami/src/konami_code_fsm.erl

```diff
@@ -56,7 +56,7 @@
                }).
 -type state() :: #state{}.
 
--define(WSD_ENABLED, 'false').
+-define(WSD_ENABLED, kapps_config:get_integer(?APP_NAME, <<"webseq_enabled">>, 'false')).
```

Comment:
sorry was half asleep while sending the pr. should i move the declaration to a central place in konami.hrl ? as both konami_code_fsm.erl and konami_transfer.erl requires it or there should be different variables for both of them. as with this variable it will either be true or false for both.